src/data/gods/abilityTypes.ts

```diff
@@ -1,11 +1,12 @@
-export type BoonType =
+export type StandardBoonType =
```

Comment:
Nit: Generally don't add `Type` to the name of a type, looks like maybe I was the first violator of that though